core/shared/src/main/scala/org/http4s/Message.scala

```diff
@@ -224,6 +224,8 @@ sealed trait Message[+F[_]] extends Media[F] { self =>
 }
 
 object Message {
+  // workaround for an apparent regression in Scala 2.13.8 - Media.InvariantOps should be resolved directly.
+  implicit def toMediaInvariantOps[F[_]](self: Media[F]): Media.InvariantOps[F] = new Media.InvariantOps[F](self)
   implicit final class InvariantOps[F[_], ThisF[f[_]] <: Message[f]](val self: ThisF[F]) {
```

Comment:
```suggestion
  final class InvariantOps[F[_], ThisF[f[_]] <: Message[f]](val self: ThisF[F]) {
```

Reply:
This might cause ambiguity issues maybe?

Reply:
If anything the ambiguity would be with the implicit constructor on `Media`, not this one, no?

Reply:
Sorry, you are right!